resolvers/index.js

```diff
@@ -1,24 +1,33 @@
+const { AuthenticationError } = require("apollo-server");
 const { Campaign } = require("../models/Campaign");
 const { User } = require("../models/User");
 const { UserAction } = require("../models/UserAction");
 
 const queryResolvers = {
 	/**
-	 * Retrive user using Cookies
-	 */
+   * Retrive user using Cookies
+   */
 	currentUser: async (root, args, context) => {
-		return { id: 1 };
+		if (!context.User.external_id) {
```

Comment:
This looks the same as my files, but I'm sure locally it looks good. Not sure if it's a GitHub issue or a linter issue (using spaces instead of tabs)

Reply:
There is indeed something weird... 
![http://recordit.co/ShPYmGMrct](http://recordit.co/ShPYmGMrct.gif)

I am taking a look to see if I can solve it right now.

Reply:
I did two things @orlando I disable prettier on my VSCode and create a new eslint config file following airbnb rules _(why airbnb? because they have semi colons _- but I just wanted something consistent for now -_)_

Reply:
I did this https://github.com/debtcollective/campaign-api/pull/10/commits/b29bfafa9da9e306982d9fc24cbe1d443b1e9113 basically with that it should work. make sure to reload your VSCode window after pull changes. 

Furthermore, the json config file has another indentation but we can address later. LGTM if the code itself feels ready.